qtox/docker/Dockerfile.host-qt

```diff
@@ -24,7 +24,6 @@ WORKDIR /work/build
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 ARG QT_VERSION=6.2.4
```

Comment:
is host qt fixed to 6.2.4 ?

Reply:
It's an arg. This is the default arg. I picked 6.2.4 because that one won't change, while the 6.8.1 will move.

Reply:
oh I see